### ISSUE TITLE
[Bug] Parsing jsx text error

### ISSUE THREAD

**Issue by fanksy** [caoccao/swc4j#5]
There is actually only one line break, but it is parsed as two line breaks

example：

```Javascript
import React from 'react';

function MyComponent() {
  return (
    <div>
      <p>Hello, this is JSX text!</p>
      <p>JSX allows us to write 
        HTML-like syntax in JavaScript.</p>
    </div>
  );
}

export default MyComponent;

```
<img width="1050" alt="Screenshot 2024-06-23 at 16 18 06" src="https://github.com/caoccao/swc4j/assets/22285815/d960b4a9-db31-4ef1-a72f-d393ca64bdf8">

**Comment by caoccao** [comment]
Please double check your test code and leave a repo that can reproduce the issue.

**Comment by fanksy** [comment]
```Java
@Test
    public void testAst() throws IOException, Swc4jCoreException {
        String script = "import React from 'react';\n" +
                "\n" +
                "function MyComponent() {\n" +
                "  return (\n" +
                "    <div>\n" +
                "      <p>Hello, this is JSX text!</p>\n" +
                "      <p>JSX allows us to write \n" +
                "        HTML-like syntax in JavaScript.</p>\n" +
                "    </div>\n" +
                "  );\n" +
                "}\n" +
                "\n" +
                "export default MyComponent;";
        Swc4j swc4j = new Swc4j();
        Swc4jTranspileOptions options = new Swc4jTranspileOptions()
                .setCaptureAst(true)
                .setMediaType(Swc4jMediaType.Jsx);
        Swc4jParseOutput parseOutput = swc4j.parse(script, options);
        assert  parseOutput != null;
    }
```

**Comment by fanksy** [comment]
Please debug and check the jsx text node.

**Comment by caoccao** [comment]
It seems to be an SWC bug. Issue [9106](https://github.com/swc-project/swc/issues/9106) was filed.

**Comment by caoccao** [comment]
It's fixed in SWC. Please wait for the patch in the next couple of releases.

**Comment by caoccao** [comment]
The fix from SWC has been picked up. The new release will be published soon.

**Comment by caoccao** [comment]
It's fixed in v0.10.0.
